pages/treatments/[uuid].js

```diff
@@ -167,9 +167,13 @@ export async function getServerSideProps({ params, req, query }) {
     const documents = treatment.documents
       ? await requestDocuments(treatment.documents, request)
       : [];
+    let treatmentId = "";
+    treatment.treatment_type === "environmental"
+      ? (treatmentId = treatment.summary)
+      : (treatmentId = treatment.treatment_term_id);
```

Comment:
Conditional expressions always return a value, which you can use here:
```
const treatmentId = treatment.treatment_type === "environmental"
    ? treatment.summary
    : treatment_term_id;
```
That way you don’t have assignments within an expression. Generally, there should be only one assignment in an assignment statement, to a single variable on the left side of an equals sign.

Reply:
i see. i'll fix it.